src/shared/components/input/IntlTelInputWithLabelFormGroup.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import IntlTelInput, { CountryData } from 'react-intl-tel-input'
+
+import useTranslator from '../../hooks/useTranslator'
+import 'react-intl-tel-input/dist/main.css'
+
+interface Props {
+  name: string
+  label?: string
+  value: string | undefined
+  isEditable?: boolean
+  onChange?: (value: string) => void
+  feedback?: string
+  isInvalid?: boolean
+}
+
+const IntlTelInputWithLabelFormGroup = (props: Props) => {
+  const { t } = useTranslator()
+  const { onChange, name, isEditable, value: val, feedback, isInvalid } = props
+  const id = `${name}IntlTelPicker`
+
+  const [valid, setValid] = useState<boolean>()
+
+  const onPhoneNumberChange = (
+    isValid: boolean,
+    value: string,
+    _selectedCountryData: CountryData,
+    fullNumber: string,
+    _extension: string,
+  ) => {
+    setValid(isValid)
+    if (onChange) {
+      valid ? onChange(fullNumber.replace(/\s/g, '')) : onChange(value)
```

Comment:
This component does in place validation. Once the user has input a valid phone number, the custom error message below the input field disappears. However, the validation that was previously taking place after the redux item is set is still in place, and that is still ultimately what decides whether the changes to the patient object are committed or not. This component makes sure the country code (+49) gets appended to the beginning of the string, at it seems like the two validation methods align.

Let me know if this makes sense or if should be somehow improved